cmd/oceantv/broadcast_hardware_machine.go

```diff
@@ -143,9 +143,13 @@ func sanatisedVoltageRecoveryTimeout(ctx *broadcastContext) int {
 
 type hardwareStopping struct {
 	*broadcastContext `json:"-"`
+	stateWithTimeoutFields
+	PreviouslyTimedOut bool
```

Comment:
I'm not a massive fan of this addition. It's basically creating a substate of hardwareStopping. We could instead look at splitting hardwareStopping into two states e.g. hardwareShuttingDown and hardwarePoweringOff, then you wouldn't need to have the weird logic around whether this previously timed out or not.

Reply:
True. I'll have a look into that.